adafruit_button.py

```diff
@@ -31,7 +37,7 @@
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_Display_Button.git"
 
 
-def _check_color(color):
+def _check_color(color: Optional[Union[int, tuple[int, int, int]]]) -> Optional[int]:
```

Comment:
We have an Alias that covers these types of Color arguments. But it's in a file that associates it with leds: https://github.com/adafruit/Adafruit_CircuitPython_Typing/blob/bd358db434d2e2c155c897d7f3c9674b04a3b67f/circuitpython_typing/led.py#L18

@tekktrik do you think it's worth making a slightly more generic Alias for Color inside of circuitpython_typing that can be used in other libraries for hex / tuple colors? Many of our libraries expect both in the context of displayio as well as LEDs.

Or alternatively is it problematic to use the typing from inside of led.py here even though this context isn't dealing with LEDs?

Reply:
I've put a note to discuss this in the weeds for the upcoming weekly meeting.

Reply:
We could always change up the way that colors are defined, potentially moving them out of `led.py` and into a new file like `color.py`.